concordium-std/src/impls.rs

```diff
@@ -1798,6 +1818,25 @@ fn parse_transfer_response_code(code: u64) -> TransferResult {
     }
 }
 
+/// Decode the the response code.
+///
+/// The response is encoded as follows.
+/// - success is encoded as 0.
+/// - failed because of missing module is 0x07_0000_0000.
+/// - failed because of module is missing contract with the same name is
+///   0x08_0000_0000.
+/// - failed because of module being an unsupported version is 0x09_0000_0000.
```

Comment:
```suggestion
/// - failure because of missing module is 0x07_0000_0000.
/// - failure because of module is missing contract with the same name is
///   0x08_0000_0000.
/// - failure because of module being an unsupported version is 0x09_0000_0000.
```